### DISCUSSION TITLE
The "sys_probs" parameter doesn't seem to be working properly

### DISCUSSION THREAD

**Discussion by 96XinHe** [deepmodeling/deepmd-kit#4346] [Q&A] [answered]
**deepmd-kit version: 3.0.0b4**

I performed a single-task training process with the DPA-2 descriptor and specified "sys_probs" for each of the 28 training sets in "training":

"training": {
    "stat_file": "./dpa2.hdf5",
    "training_data": {
      "systems": [
...............................
      ],
      "batch_size": 1,
      "sys_probs": [0.03,0.05,0.05,0.03,0.03,0.03,0.05,0.05,0.03,0.03,0.03,0.03,0.05,0.03,0.03,0.03,0.03,0.03,0.03,0.05,0.03,0.03,0.03,0.05,0.03,0.03,0.05,0.03],
      "_comment": "that's all"
    },

But it looks like this parameter doesn't work properly, and the probabilities for each training set in the output log are still assigned by default:
![图片](https://github.com/user-attachments/assets/0ae3f92b-9f9f-42ab-b496-cc96fe7a5388)

The “sys_probs” parameter is also correctly recognized in the auto-generated input_v2_compat.json file

**Comment by wanghan-iapcm** [comment]
which back-end did you use?  or plz provide your command line

**Reply by 96XinHe** [reply]
the command line is:
`dp --pt train input_torch_medium.json`

**Comment by iProzd** [comment, marked as answer]
`sys_probs` was not used in pt, because its priority was lower than that of `auto_prob`, while `auto_prob` always has its default values.
Fixed in https://github.com/deepmodeling/deepmd-kit/pull/4353
